spec/requests/api/v1/users_spec.rb

```diff
@@ -5,23 +5,6 @@
   let(:Authorization) { "Bearer #{generate_jwt_token(user)}" }
 
   path '/api/v1/users' do
-    get('list users') do
```

Comment:
I think this method we need for a blog (maybe for a mention) or QA

Reply:
Now we don't have this functional.